reddit_service_websockets/socketserver.py

```diff
@@ -227,6 +228,11 @@ def _quiesce(self, environ, bypass_auth=False):
                                    self._shed_connections,
                                    [conns.pop() for j in xrange(num_conns)])
 
+            # Terminate the service after shedding
+            termination_delay_secs = 10
+            gevent.spawn_later(shed_delay_secs + cur_iter_sec + \
```

Comment:
i don't think the \ is necessary since you're in a () expression which can happily span lines